lib/browser/property-descriptor.ts

```diff
@@ -23,24 +230,32 @@ export function propertyDescriptorPatch(_global: any) {
   if (canPatchViaPropertyDescriptor()) {
     // for browsers that we can patch the descriptor:  Chrome & Firefox
     if (isBrowser) {
-      patchOnProperties(window, eventNames.concat(['resize']));
+      patchOnProperties(window, eventNames, Object.getPrototypeOf(window));
```

Comment:
The fact that we are getting all of the properties of the window, does not seem right. Why can't we have a fixed list here as well?